clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/model/SecurityGroupLookupFactory.groovy

```diff
@@ -40,7 +42,7 @@ class SecurityGroupLookupFactory {
     private final String region
     private final ImmutableSet<NetflixAmazonCredentials> accounts
 
-    private final Map<String, List<SecurityGroup>> securityGroupsByAccount = [:]
+    private final Map<String, SecurityGroup> securityGroupByName = [:]
```

Comment:
Honestly not sure the importance of keep an instance lifetime cache of security groups.